README.md

```diff
@@ -766,6 +766,24 @@ let darker = color.darker(amount: 0.5)
 let darker = color.darker()
 ```
 
+### CGRect extension
+
+Get the origin's x and y coordinates:
+
+```swift
+aRect.x // instead of aRect.origin.x
+aRect.y // instead of aRect.origin.y
+```
+
+Change one property of a `CGRect`:
+
+```swift
+let rect = CGRect(x: 10, y: 20, width: 30, height: 40)
+let widerRect = rect.with(width: 100)
+let tallerRect = rect.with(height: 100)
+let rectAtAnotherPosition = rect.with(x: 100).with(y: 200)
```

Comment:
What do you mean by "result printed once changed"? I didn't have any print statements.